### PR TITLE
Corrected location of files and default exports for functional components

### PR DESCRIPTION
**Pull request by davidchuang5**
### Description

- Removed frontend files from the backend folder
- Moved Sidebar.tsx into it's own folder
- Removed redundant export of Sidebar and ProcessLogs component from files
- Added default export to Sidebar and ProcessLogs functional components
- Removed unused React import in Logs.tsx
- Changed import statement of Sidebar in Logs.tsx
- Fixed typos

### CODE REVIEW

File: ui/src/pages/Logs/Logs.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ import {
 } from '@mui/material';
 import SearchIcon from '@mui/icons-material/Search';
 import * as React from 'react';
-import { SideBar } from '../../components/Sidebar';
+import { SideBar } from '../../components/Sidebar/Sidebar';
```
Comment (scotthallock) [inline ui/src/pages/Logs/Logs.tsx]: Could you make the SideBar component the default export to stay consistent with our other components?

File: ui/src/components/Sidebar/Sidebar.tsx [hunk 2]
```diff
@@ -86,14 +86,17 @@ export const SideBar: React.FC = () => {
   /******** Type filter code ************/
   // Function for checking all type boxes
   const checkAllTypes = (event: React.ChangeEvent<HTMLInputElement>) => {
+    console.log('All types box checked');
```
Comment (scotthallock) [inline ui/src/components/Sidebar/Sidebar.tsx:89]: Remove the console logs

**Review by scotthallock — APPROVED** [review]

**Review by leejun07 — APPROVED** [review]
Looks much cleaner!

**Review by patrickv77 — APPROVED** [review]
Thank you for removing the additional files and coordinating with Scott to have consistency in export statements!

**Pull request merged**